ui/src/static/env.js

```diff
@@ -1,8 +1,8 @@
 window.env = {
   APPLICATION_NAME: 'Taskcluster',
   TASKCLUSTER_ROOT_URL: 'https://taskcluster.net',
```

Comment:
The typical testing / fake root URL is `https://tc.example.com` -- that might be a good choice here.